docs/developer_guide/self-hosted.md

```diff
@@ -168,4 +168,76 @@ Once your droplet has been created, open the console to ssh to the server, follo
+      - **RS:** send `save-cronjob.sh` to remote server
+        - NOTE : Replace the "$REMOTE_IP" with the actual ip
+        - `rsync -avzh script/save-cronjob.sh root@"$REMOTE_IP"167.71.33.149:save-cronjob.sh `
+      - **RS:** ssh into the remote server 
+        - **RS** ssh `root@"$REMOTE_IP"`
+        - **RS:** Do `ls` to check if `athens-cli.jar` and `save-cronjob.sh` files are present on the remote server, if not please 
+          go through the above steps again and if the problem still exists please ask us in the Discord.
+        - Set up an hourly cronjob for the Save (Backup)
+          - Run `crontab -e`
+          - Add the following line to the file that is opened after the above command:
+            - `0 */1 * * * ~/save-cronjob.sh`
+            - The above command will run the `save-cronjob.sh` script every hour and will save your fluree ledger (current source of truth)
+              to `/var/lib/athens/backups/` folder.
+            - To change the frequency of the backups, change the cronjob parameters accordingly. You can see the docs explaining cronjob command parameters by 
+              running the command `crontab -e`
+        - NOTE: After a couple of hours after setting up this cronjob please do check if the backups are happening by going to `/var/lib/athens/backups/`
+
+### Load (Restore) your server from one of the backups
+ 
+ - Load (Restore) assumes that you have gone through the Save (Backup) process.
+ - **RS** ssh into the remote server `ssh root@"$REMOTE_IP"`
+   - Stop the Athens server docker `docker-compose stop athens`
+   - Find the file that you want to restore from, if you did not change the default save directory then you can find all
+     the save (Backed up) files in `/var/lib/athens/backups`
+   - Load the file to Athens server `yarn cli:load -f "$FILENAME"` (Replace `"$FILENAME` with the correct filename)
+   - Follow the instructions outputted by the above command.
+   - Restart the Athens server `docker-compose start athens`
+   - If you are not able to load (Restore) the file please go through the steps again and if the problem still exists then
+     let us know in our Discord server.
+
+### Get the Saved (Backed up) to your local machine
+ 
+ - Here we assume that you have gone through the Save (Backup) process
+ - **RS** ssh into the remote server
+   - `ssh root@IP`
+ - goto `/var/lib/athens/backups` and find the name of file you want to get
+     - `cd /var/lib/athens/backups`
+     - `ls` : to see all the files and copy the filename
+ - log out of the serve: `ctrl-d`
+ - Get the server log to local machine
+     - NOTE : 
+       - Replace the "$REMOTE_IP" with the actual ip also 
+       - Replace the "$FILENAME" with the actual filename
+       - Replace the "$LOCAL_DIR" with the path of dir where you want to copy the save (Backup) file in your local machine.
+     - `rsync -avzh root@"$REMOTE_IP"46.101.61.240:/var/lib/athens/backups/"$FILENAME" "$LOCAL_DIR"`
```

Comment:
Actually `scp` is easier, and we already assume that user has `ssh`.

Reply:
yeah you are right